### PR TITLE
init python dimensionality

### LINKED ISSUE #2: Multi-dimensionality in Python
Mirroring #1 in Python to get familiar with the programming language python

### PR DESCRIPTION
**Pull request by capital-G**
Closes #2

### CODE REVIEW

File: 00_basics/py_dimensions.ipynb [hunk 1]
```diff
@@ -0,0 +1,740 @@
+{
```
Comment (telephon) [inline 00_basics/py_dimensions.ipynb:1]: <p>an unwritten</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (telephon) [inline 00_basics/py_dimensions.ipynb:1]: <p>I'd move $= (\sqrt(x))^2$ to the end, not to presuppose what is to be achieved.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (telephon) [inline 00_basics/py_dimensions.ipynb:1]: <p>– which is not how we defined matrix multiplication</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (telephon) [inline 00_basics/py_dimensions.ipynb:1]: <p>btw. is it not a convention in python to have spaces between operators and operands?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (telephon) [inline 00_basics/py_dimensions.ipynb:1]: <p>to the large the number of third-party libraries</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (telephon) [inline 00_basics/py_dimensions.ipynb:1]: <p>let's</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

**Comment by capital-G** [comment]
Thanks for the remarks - I really need to learn the proper use of `let's`.
I hope everything is fixed now - if so please merge.

**Comment by telephon** [comment]
>  I really need to learn the proper use of `let's`

It mathematics, it's useful sometimes

**Pull request merged**